serialization/README.md

```diff
@@ -4,7 +4,7 @@
 
 For information on what can be included in an SPDX serialization and how they
 are structured, please refer to the
-[Serialization information](https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/serializations.md)
+[Model and serializations](https://github.com/spdx/spdx-spec/blob/develop/docs/serializations.md)
```

Comment:
Why the change in title to include Model?

Reply:
I took the title from the document at the URL.  It is Chapter 6 of the spec.

Reply:
I am now update the link to the specific section, retain the previous section reference, and update the text that it is a "Serialization information" section within a "Model and serializations" chapter.